code/datums/syndicate_buylist.dm

```diff
@@ -468,6 +468,15 @@ This is basically useless for anyone but miners.
 	job = list("Clown")
 	blockedmode = list(/datum/game_mode/spy)
 
+/datum/syndicate_buylist/traitor/boomboots
+	name = "Boom Boots"
+	item = /obj/item/clothing/shoes/cowboy/boom
+	cost = 12
+	vr_allowed = 0
+	desc = "A funny-looking car designed for circus events. Seats 30, very roomy! Comes with an extra set of clown clothes."
```

Comment:
This seems wrong. Clown car description?

Reply:
lol... oops